pages/textbook-demo/learning-paths/introduction-course.vue

```diff
@@ -10,6 +10,9 @@
 <script lang="ts">
 import { Component } from 'vue-property-decorator'
 import QiskitPage from '~/components/logic/QiskitPage.vue'
+import { GeneralLink } from '~/constants/appLinks'
+
+const segmentActionBase = 'introduction-course > external-recommended-readings >'
```

Comment:
If you move this line after [line 26](https://github.com/Qiskit/qiskit.org/pull/1563/files#diff-c0470fa3ad319c3ec454915a8dae5f0f5bc2def97b395e1078e3329dea09756aR26), you can reuse the `routeName` and we have the constant close to where it is used

Reply:
Good idea! I'll change it. 🙂